Dockerfile.frontend

```diff
@@ -7,6 +7,4 @@ COPY package.json package.json
 COPY yarn.lock yarn.lock
 RUN yarn install
 
-COPY . .
```

Comment:
Tried this as an attempt to fix https://github.com/18F/e-QIP-prototype/pull/418#issuecomment-397022500 - turned out to be useful regardless.